routing/missioncontrol.go

```diff
@@ -121,6 +121,10 @@ type MissionControl struct {
 // MissionControlConfig defines parameters that control mission control
 // behaviour.
 type MissionControlConfig struct {
+	// callback is a function that is called whenever the mission control
```

Comment:
nit: UpdateEstimator is a ... instead of "callback" in the comment

Reply:
I changed the name of the function and did not update the comment ... just fixed